scripts/nginx/readarr.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# Nginx conf for Readarr
+# Bakerboy448 for swizzin 2020
+# Based on Radarr v3 installationn by flying sausages
+master=$(cut -d: -f1 < /root/.master.info)
+
+cat > /etc/nginx/apps/readarr.conf << readarr
+location /readarr {
+  proxy_pass        http://127.0.0.1:8787/readarr;
+  proxy_set_header Host \$proxy_host;
+  proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
+  proxy_set_header X-Forwarded-Proto \$scheme;
+  proxy_redirect off;
+  auth_basic "What's the password?";
+  auth_basic_user_file /etc/htpasswd.d/htpasswd.${master};
+
+  proxy_http_version 1.1;
+  proxy_set_header Upgrade \$http_upgrade;
+  proxy_set_header Connection \$http_connection;
+}
+readarr
+
+isactive=$(systemctl is-active readarr)
+
+if [[ $isactive == "active" ]]; then
+    echo_log_only "Stopping readarr"
+    systemctl stop readarr
+fi
+user=$(grep User /etc/systemd/system/readarr.service | cut -d= -f2)
+echo_log_only "readarr user detected as $user"
+apikey=$(grep -oPm1 "(?<=<ApiKey>)[^<]+" /home/"$user"/.config/readarr/config.xml)
+echo_log_only "Apikey = $apikey" >> "$log"
+#TODO change Branch whenever that becomes relevant
+cat > /home/"$user"/.config/readarr/config.xml << readarr
+# Readarr is Pre-Alpha, set logs to trace
+<Config>
+  <LogLevel>trace</LogLevel>
+  <UpdateMechanism>BuiltIn</UpdateMechanism>
+  <BindAddress>127.0.0.1</BindAddress>
+  <Port>8787</Port>
```

Comment:
Might wanna change this if it conflicts as mentioned above. Will the defaults change upstream?

Reply:
as far as I know the guys have no plans to change from 8787